lib/generators/draft/resource/resource_generator.rb

```diff
@@ -61,27 +61,27 @@ def generate_specs
 
   private
 
-    def golden_seven_routes
+    def golden_five_routes
       log :route, "RESTful routes"
 
       route <<-RUBY.gsub(/^      /, "")
 
         # Routes for the #{singular_table_name.humanize} resource:
 
         # CREATE
-        get("/#{plural_table_name}/new", { :controller => "#{plural_table_name}", :action => "new_form" })
-        #{skip_post? ? "get" : "post"}("/create_#{singular_table_name}", { :controller => "#{plural_table_name}", :action => "create_row" })
-
+        match("/post_#{singular_table_name}", { :controller => "#{plural_table_name}", :action => "create", :via => "#{skip_post? ? "get" : "post"}"})
```

Comment:
Hmm, looking at it with the `:via` changed makes me not like it as much — it seems a bit redundant. What do you think about `/insert_#{singular_table_name}`?